style.css

```diff
@@ -81,9 +127,285 @@ li {
   margin: 0 8px;
 }
 
-button {
+@media (min-width: 992px) {
+  .header-nav-phone {
+    display: none;
+  }
+
+  .header-title {
+    padding-top: 3em;
+  }
+
+  .header-nav-desktop {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    padding-top: 1em;
+    margin: 0;
+  }
+
+  .header-nav-desktop li {
+    list-style: none;
+    padding: 0;
+    margin: 1em;
+    font-size: 14px;
+    font-weight: bold;
+    color: #42526e;
+    cursor: pointer;
+  }
+
+  .intro {
+    background: #fff url(./assets/Header-llustration-desktop.svg) no-repeat;
+    background-size: 100%;
+    padding-bottom: 8em;
+  }
+}
+
+@media (min-width: 1024px) {
+  .intro {
+    padding-bottom: 15em;
+  }
+}
+
+/* Works Section */
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 200px repeat(5, 6fr);
+  text-align: center;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.works h1:first-child {
+  padding: 1em;
+}
+
+.works h1:first-child::after {
+  content: '';
+  height: 4px;
+  width: 48px;
+  margin: 0 auto;
+  display: block;
+  position: relative;
+  top: 20px;
+  z-index: -1;
+  background: var(--color-primary);
+  border-radius: 8px;
+}
+
+.card-work {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 0;
+  margin: 1em auto;
+  width: 327px;
+  height: 474px;
+  border: 1px lightgrey solid;
+  border-radius: 5px;
+  transition: box-shadow 0.4s ease-out;
+}
```

Comment:
- [ ] Please use the background for the card that match [Figma desing](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) 
Figma design :- 
![image](https://user-images.githubusercontent.com/74744787/159078036-205fdc17-e040-4495-b3dc-e809314825b4.png)
Your design :- 
![image](https://user-images.githubusercontent.com/74744787/159078214-989207dd-a550-48c5-8a68-afd25fd1e169.png)